packages/react-components/src/localization/locales/en-US/strings.json

```diff
@@ -123,6 +123,11 @@
     "tooltipOnContent": "Lower Hand",
     "tooltipOffContent": "Raise Hand"
   },
+  "reactionButton": {
+    "label": "React",
+    "tooltipDisabledContent": "Reaction action is disabled",
+    "tooltipContent": "Reaction action"
```

Comment:
Are the strings here matching with teams? I think we should make sure the different labels are appropriate to match there

Reply:
Thanks. Checking and updating accordingly.